common/config/api.go

```diff
@@ -14,6 +15,11 @@ func APIFlags() []cli.Flag {
 			Value:  9090,
 			EnvVar: ENVVAR_PORT,
 		},
+		cli.DurationFlag{
+			Name:   FLAG_JOB_EXPIRY,
+			Value:  time.Hour * 24,
```

Comment:
This should have a default variable in `common.go` in this directory.

Reply:
Like this in  common/config/common.go

const (
	ENVVAR_JOB_EXPIRY           = time.Hour * 24

Reply:
Yup, the example you have looks good. 

Good question. It doesn't make sense for **all** defaults to exist in the `config` package - for example, the default aws iam policies obviously should live in the `provider/aws` package. However, I agree that it would make more sense for things like the default port to live here. To be honest, I figured it wasn't worth putting up the fight when I was reviewing that PR. Welcome to you making that change though.